docroot/themes/humsci/su_humsci_theme/scss/components/molecules/_hover-menu.scss

```diff
@@ -206,6 +206,7 @@
         .ui-widget {
           background: $color-white;
 
+          @include margin(1rem null null);
```

Comment:
This changes the look when a top menu item is a real link. see screenshots: 
![screen shot 2018-09-05 at 11 41 48 am](https://user-images.githubusercontent.com/7185045/45114030-c2bbbd00-b100-11e8-8738-7fb355299971.png)
![screen shot 2018-09-05 at 11 41 58 am](https://user-images.githubusercontent.com/7185045/45114031-c3545380-b100-11e8-8a5f-37c62e96e3ff.png)